bears/vcs/mercurial/HgCommitBear.py

```diff
@@ -0,0 +1,60 @@
+import re
+import shutil
+import os
+
+from urllib.parse import urlparse
+from coala_utils.ContextManagers import change_directory
+from coalib.misc.Shell import run_shell_command
+from bears.vcs.CommitBear import _CommitBear
+
+
+class HgCommitBear(_CommitBear):
+    LANGUAGES = {'Hg'}
+    SUPPORTED_HOST_KEYWORD_REGEX = {
+        'bitbucket': (r'[Cc]los(?:e[sd]?|ing)'
```

Comment:
this can go in `_CommitBear`, as bitbucket supports git, and thus git commit bear would need this too for bitbucket support.

A hg remote will never be gitlab or github , so those items being in `SUPPORTED_HOST_KEYWORD_REGEX` is not going to cause a problem.